Recrutify/Recrutify.Host/Controllers/ProjectsController.cs

```diff
@@ -87,5 +90,21 @@ public async Task<ActionResult<ProjectDTO>> GetByIdAsync(Guid id)
 
             return Ok(project);
         }
+
+        [Authorize(Policy = Constants.Policies.AllAccessPolicy)]
+        [HttpGet("primary_skills_and_staff")]
+        public async Task<ActionResult<PrimarySkillsAndStaffDTO>> PrimarySkillsAndStaff()
+        {
+            var users = await _userService.GetByRoleAsync();
+
+            return Ok(new PrimarySkillsAndStaffDTO()
```

Comment:
move logic of creating dto to service - alse use not [] but TryGetValue, cause if you for example don't have managers it will be exception in moment users[Role.Manager]